### PR TITLE
Adding MSXIRC for the first time

### PR DESCRIPTION
**Pull request by Fubukimaru**
Hello!

I have created a package for MSXIRC version v1.1.

I hope it works correctly!

### CODE REVIEW

**Comment by Fubukimaru** [comment]
I have added the license as "unknown" as we still don't have it under any license.

File: packages/MSXIRC.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+name: 'MSXIRC'
+version: '1.1'
+release: 1
+summary: 'MSXDOS IRC client'
+author: 'Ptero and Oduvaldo Pavan Junior'
+package_author: 'Fubukimaru'
+license: 'Unknown'
+category: 'Internet'
+system: 'MSX'
```
Comment (fr3nd) [inline packages/MSXIRC.yaml:10]: I read in the documentation this requires at least MSX2, so this should probably be changed, right?
```suggestion
system: 'MSX2'
```

The list of systems can be found here: https://msxhub.com/list_systems

**Review by fr3nd — COMMENTED** [review]
Hi! Thanks for your contribution! Having a nice IRC client for the MSX is really cool. I've just added one comment to your changes. Could you review it? Everything else seems fine to me 🔝

**Comment by Fubukimaru** [comment]
Hello fr3nd!

Sorry, I forgot to update that! Change accepted :)

**Comment by fr3nd** [comment]
Thanks! The package will be available in few minutes

**Pull request merged**